config/session.php

```diff
@@ -53,7 +53,7 @@
     |
     */
 
-    'encrypt' => false,
+    'encrypt' => true,
```

Comment:
Not sure at all lol I just saw you both change it, thought it was because the security issue :stuck_out_tongue_winking_eye: